src/features/nft/nftModals/bidAuctionModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import { Button } from '../../../common/components/Buttons'
+import { InputNumberFormat } from '../../../common/components/Inputs'
+
+type TBidAutionType = 'MakeOffer' | 'Bid'
+
+export type TBidAuctionModal = {
+  type: TBidAutionType
+  username?: string
+  isOpen: boolean
+  handleClose?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const BidAuctionModal = ({
+  type,
+  username,
+  isOpen,
+  handleClose,
+}: TBidAuctionModal): JSX.Element => {
+  const [bid, setBid] = React.useState<string>('12950')
+  const handleBid = (event: React.ChangeEvent<HTMLInputElement>) =>
+    setBid(event.target.value)
+
+  return (
+    <Modal isOpen={isOpen} handleClose={handleClose} size='lg'>
+      <h2 className='mb-6'>
+        {type === 'MakeOffer' ? 'Make an offer' : 'Your bid'}
+      </h2>
+      <p className='mb-6'>
+        You are place a bid for{' '}
+        <span className='font-medium text-blue-3f'>
+          Super Nfty Floating Head Professional
+        </span>{' '}
+        by <span className='font-medium text-blue-3f'>{username}</span>
+      </p>
+      <div className='mb-6'>
+        <InputNumberFormat
+          hint='Your bid must be higher then current one'
+          type='text'
+          label='Your bid'
+          value={bid}
+          onChange={handleBid}
+          append={<span className='text-gray-a0'>PSL</span>}
```

Comment:
Can we get the name(PSL) from `currencyName` because we have a testnet and the `PSL` will change to `PLS`. Please also update for below sections

Reply:
![image](https://user-images.githubusercontent.com/49214944/122414758-ab83ac80-cf87-11eb-8927-72a84e90c10a.png)
I followed the design. If this is wrong, we have to confirm this with Jeff.